src/components/calculator.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+/*eslint-disable */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice. Instead of disabling `eslint` totally, I think for this case, it is allowed to disable `react/prefer-stateless-function` like so `/* eslint-disable react/prefer-stateless-function */`. Please kindly fix that.

Reply:
Thank you very much @Mihndim2020.
 
I wanted to do exactly that but I didn't know-how.
I will make the above changes.